packages/dds/tree/src/test/simple-tree/api/schemaFactoryRecursive.spec.ts

```diff
@@ -715,4 +756,130 @@ describe("SchemaFactory Recursive methods", () => {
 		const r = hydrate(Root, { r: new ArrayRecursive([]) });
 		assert.deepEqual([...r.r], []);
 	});
+
+	/**
+	 * These cases are anti-patterns which are documented not to work, but still might get used my someone.
+	 * SchemaFactory documents the non explicit sub-classing pattern (POJO mode) is not supported for recursive schema.
+	 * Additionally it documents this pattern is not supported for Explicitly named Maps and Arrays.
+	 *
+	 * These tests violate both of these statements, seeing which version currently compile and which ones do not.
+	 * Having these tests in place will help know when.if be further break cases like these to adi in writing helpful changesets for impacted customers.
```

Comment:
The "when.if" part seems to be accidental.

Reply:
```suggestion
	 * Having these tests in place will help discover when/if further changes break cases like these to aid in writing helpful changesets for impacted customers using these unsupported patterns on accident.
```